README.md

```diff
@@ -4,7 +4,31 @@ Next Gen docos using VuePress.
 
 ## Local development
 
+### Prerequisites
+
+1. `NodeJs` installed (see [installation guide](https://nodejs.org/en/download//)).
```

Comment:
NodeJS

Reply:
Removed in https://github.com/EventStore/documentation/pull/163/commits/b48cca4b907699b1cb5bb21347f39fd0ed4855b3.